workspaces/ui-v2/src/optic-components/common/EndpointName.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
Moved from `/src/optic-components/documentation/EndpointName.tsx` - this is effectively a "common" component that we can use wherever we need to render an endpoint name